lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -147,7 +147,7 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
     config: (version) => ({
       datasource: FlutterVersionDatasource.id,
       // asdf-flutter plugin supports channel on version suffix.
-      currentValue: version.replace(/-(stable|beta|dev)$/, ''),
+      currentValue: version.replace(regEx(/-(stable|beta|dev)$/), ''),
```

Comment:
you need to import from `utils/regex`

Reply:
Yes, I fixed it on this.

https://github.com/renovatebot/renovate/commit/6a8abfb0f0e13cc9996eccb119c6741b1c6a4add